<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Acknext MIEP *FLOAT*</string>
  </property>
  <widget class="QAcknextWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>739</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionImport_model"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionOptions"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionScene_View"/>
   <addaction name="actionLight_View"/>
   <addaction name="actionNormal_View"/>
   <addaction name="separator"/>
   <addaction name="actionWireframe"/>
   <addaction name="separator"/>
   <addaction name="actionShow_Mesh"/>
   <addaction name="actionShow_Skelton"/>
   <addaction name="actionShow_normals"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="fileToolbar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionImport_model"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../../resource/ackui.qrc">
     <normaloff>:/icons/folder-open.png</normaloff>:/icons/folder-open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../../resource/ackui.qrc">
     <normaloff>:/icons/content-save.png</normaloff>:/icons/content-save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="icon">
    <iconset resource="../../resource/ackui.qrc">
     <normaloff>:/icons/settings.png</normaloff>:/icons/settings.png</iconset>
   </property>
   <property name="text">
    <string>O&amp;ptions</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionNormal_View">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Normal View</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
  <action name="actionLight_View">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Light View</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionScene_View">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Scene View</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionWireframe">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resource/ackui.qrc">
     <normaloff>:/icons/wireframe.png</normaloff>:/icons/wireframe.png</iconset>
   </property>
   <property name="text">
    <string>Wireframe</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+4</string>
   </property>
  </action>
  <action name="actionShow_Mesh">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resource/ackui.qrc">
     <normaloff>:/icons/cube.png</normaloff>:/icons/cube.png</iconset>
   </property>
   <property name="text">
    <string>Show Mesh</string>
   </property>
  </action>
  <action name="actionShow_Skelton">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resource/ackui.qrc">
     <normaloff>:/icons/skeleton.png</normaloff>:/icons/skeleton.png</iconset>
   </property>
   <property name="text">
    <string>Show Skelton</string>
   </property>
  </action>
  <action name="actionImport_model">
   <property name="icon">
    <iconset resource="../../resource/ackui.qrc">
     <normaloff>:/icons/file-import.png</normaloff>:/icons/file-import.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Import modelâ€¦</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="actionShow_normals">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resource/ackui.qrc">
     <normaloff>:/icons/show-normal.png</normaloff>:/icons/show-normal.png</iconset>
   </property>
   <property name="text">
    <string>Show normals</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QAcknextWidget</class>
   <extends>QWidget</extends>
   <header location="global">qacknextwidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resource/ackui.qrc"/>
 </resources>
 <connections/>
</ui>
